packages/ui/src/components/Navigation/components/LanguagesDropdown.tsx

```diff
@@ -51,38 +28,38 @@ export function LanguagesDropdown({
             label={<GlobeAltIcon className="w-6 h-6" />}
         >
             <Dropdown.Group>
-                {displayedLanguages.map((language, index) => {
-                    const translationLink = hasError
-                        ? '/'
-                        : getTranslationUrl(
-                              LocaleObject.fromAnyCode(locale),
-                              currentCategory,
-                              currentStory,
-                          );
-
-                    const link =
-                        currentStory && translationLink !== '/'
-                            ? translationLink
-                            : `/${language.code}${translationLink}`;
-
-                    return (
-                        <Dropdown.Item
-                            className={twMerge(
-                                'border-b border-gray-200',
-                                index === displayedLanguages.length - 1 && `border-b-0`,
+                {displayedOptions.map(({ code, name, href }, index) => (
+                    <Dropdown.Item
+                        className={twMerge(
+                            'border-b border-gray-200',
+                            index === displayedOptions.length - 1 && `border-b-0`,
```

Comment:
https://tailwindcss.com/docs/hover-focus-and-other-states#first-last-odd-and-even

```suggestion
                            `last:border-b-0`,
```